internal/cli/atlas/clusters/clusters.go

```diff
@@ -93,18 +93,19 @@ func Builder() *cobra.Command {
 
 func addTags(out *atlasv2.AdvancedClusterDescription, tags map[string]string) {
 	if len(tags) > 0 {
-		out.Tags = []atlasv2.ResourceTag{}
-	}
-	for k, v := range tags {
-		if k == "" || v == "" {
-			continue
-		}
-		key, value := k, v
-		tag := atlasv2.ResourceTag{
-			Key:   &key,
-			Value: &value,
+		t := []atlasv2.ResourceTag{}
```

Comment:
the recommended way to initialize a slice
```suggestion
		vt []atlasv2.ResourceTag{}
```

Reply:
done, thanks